scripts/start_quickstart_argilla.sh

```diff
@@ -11,13 +11,13 @@ sleep 30
 echo "Running database migrations"
 python3.9 -m argilla database migrate
 
-echo "Creating admin user"
+echo "Creating owner user"
 python3.9 -m argilla users create \
   --first-name "Admin" \
```

Comment:
Maybe here you can put "Admin owner" or "Owner admin". not sure

Reply:
Both look confusing indeed 🤣 Maybe we can just refactor everything in the next release to rename all the admin occurrences to owner, and create one user of each role? LMWYT

Reply:
"Owner of the Admins" should do the trick 🤣

More seriously, i agree with your proposal for the next release